torch/_inductor/codecache.py

```diff
@@ -36,22 +36,25 @@
     Any,
     Callable,
     cast,
-    Counter,
     Dict,
     Generator,
     List,
     NoReturn,
     Optional,
     Sequence,
-    Set,
     Tuple,
     TYPE_CHECKING,
     TypeVar,
     Union,
 )
-from typing_extensions import TypeAlias
 
 import torch
+
+# WARNING: Do not directly import has_frozen_params, it is monkeypatched in
+# python test/inductor/test_codecache.py
+# TestFxGraphCache.test_constant_handling_device_cpu
+# TODO: Why are we monkeypatching it......
+import torch._inductor.output_code as output_code
```

Comment:
@masnesral you added the test in question, is there really no way to do it without monkeypatching?

Reply:
I can get the testing I want more directly from another test in that same file: https://github.com/pytorch/pytorch/pull/141898